src/Exception/MultipartUploadException.php

```diff
@@ -21,8 +24,10 @@ public function __construct(UploadState $state, $prev = null) {
             /** @var $error AwsException */
             foreach ($prev as $part => $error) {
                 $msg .= "- Part {$part}: " . $error->getMessage(). "\n";
+                $this->collectPathInfo($error->getCommand());
             }
         } elseif ($prev instanceof AwsException) {
```

Comment:
What happens if this if/else branch and the branch above do not evaluate? The filePath value is not set. What exceptions would come through here that aren't an array or AwsException? Any random exception thrown in a promise I guess?